src/_gettsim_tests/test_data/erwerbsm_rente/2001/teilw_erwerbsgemindert_birthyear_1980_claim.yaml

```diff
@@ -0,0 +1,24 @@
+---
+info:
+  note: ''
+  source: Own calculations
+inputs:
+  assumed:
+    entgeltp_ost_erwerbsm_rente:
+      - 44.15686275
+    entgeltp_west_erwerbsm_rente:
+      - 0.0
+    erwerbsm_rente_zugangsfaktor:
+      - 0.892
+    ges_rente_vorauss_erwerbsm:
+      - true
+    rentenartfaktor:
+      - 0.5
+  provided:
```

Comment:
```suggestion
```
Just get rid of the `provided` keys everywhere, unless the source is based on some external reference. The idea of `provided` is that this is what is published somewhere (e.g., some calculator of the DRV). Very often, the inputs of some calculator on the web there won't be enough for GETTSIM's functions, hence we add the `assumed` for additional input columns. If we just made up everything ourselves, it should just  be `assumed`